docs/getting-started/cli-guide.md

```diff
@@ -61,24 +61,24 @@ The general flow for using the CLI is as follows:
 * **Load Program**: The first step is to load the eBPF Program (or Programs) using the
   `bpfman load` command.
   The programs can be from a locally built eBPF program (.o file) or an eBPF program
-  packaged in a OCI container image from a given registry.
+  packaged in an OCI container image from a given registry.
   Once the command completes successfully, the eBPF programs are loaded in kernel memory,
-  but have not been attached to any hook points yet.
+  but have not been attached to any hook points, yet.
 * **Attach Program**: The next step is to attach a loaded eBPF Program to a hook point
   using the `bpfman attach` command.
   Each program type (kprobe, tc, tracepoint, xdp, etc) has unique hook points and unique
   configuration data which is provided with the attach command.
-  Once attached, the eBPF Program will be called if it's hook point is triggered.
+  Once attached, the eBPF Program will be called if its hook point is triggered
```

Comment:
```suggestion
  Once attached, the eBPF Program will be called when its hook point is triggered.
```

Reply:
I love how I managed to fix a ' and then removed the full stop ;)